docs/development/language-features.md

```diff
@@ -159,17 +161,17 @@ import { suffixInflection } from "../language-transforms.js";
 export const englishTransforms = {
   language: "en",
   conditions: {},
-  transforms: [
-    {
+  transforms: {
+    plural: {
```

Comment:
I think Prettier or Husky is automatically removing the brackets around `plural` every time i add them